### DISCUSSION TITLE
Login for Start when php doesn't send mail

### DISCUSSION THREAD

**Discussion by freddey** [lukasboc/onepagercms#112] [General]
Hi I installed onepagecms but I cannot get in. I do not get a mail after I create a user. It is probably because php doesnot allow it in the webspace Im using. I use Limacity. Do you have any Idea to get a login for the first login?
Hope to hear from you. Fred

**Comment by lungkao** [comment]
me too

**Reply by lukasboc** [reply]
You can see a possible solution for this problem here: https://github.com/lukasboc/onepagercms/issues/113#issuecomment-1036060819
Although I guess that if the regular registration does not work, you will face more problems using the application. I would rather try a different hosting service if I was you.

Greetings from Oldenburg
Lukas
